### ISSUE TITLE
Why is Tax-Calculator showing lower wages and salaries than expected?

### ISSUE THREAD

**Issue by andersonfrailey** [open-source-economics/Tax-Calculator#1156]
Over in PR #1105 a discussion was started about why total Wages and Salaries in TaxCalc are lower than they should be. This issue summarizes the main points made in that PR and serves as a location for further discussion on that topic.

Before and after #1105, wages and salaries produced by TaxCalc were bellow CBO's latest projections, as can be seen here:

| Year | CBO Projections | Diff. After | Diff. before | Change in Error | 
|------|-----------------|-----------|-----------|-----------------| 
| 2016 | 8121            | -636.70   | -658.70   | -22.00          | 
| 2017 | 8470            | -659.80   | -679.10   | -19.30          | 
| 2018 | 8828            | -699.90   | -710.00   | -10.10          | 
| 2019 | 9154            | -730.40   | -725.10   | 5.30            | 
| 2020 | 9482            | -754.10   | -732.50   | 21.60           | 
| 2021 | 9843            | -777.90   | -740.80   | 37.10           | 
| 2022 | 10230           | -808.10   | -752.30   | 55.80           | 
| 2023 | 10634           | -839.70   | -760.70   | 79.00           | 
| 2024 | 11053           | -873.20   | -768.40   | 104.80          | 
| 2025 | 11488           | -907.50   | -776.10   | 131.40          | 
| 2026 | 11940           | -943.50   | -783.10   | 160.40          | 

I speculated that part of the difference might be due to total wages and salaries in the 2009 PUF being lower than CBO estimates for 2009 and that gap never closing:

>I believe what might explain part of the difference is that in the 2009 matched PUF we are using, there is a total of $5.848 trillion in wages and salaries, whereas in their [2010 economic outlook report](https://www.cbo.gov/sites/default/files/111th-congress-2009-2010/reports/01-26-outlook.pdf), CBO estimates 2009 wages and salaries at $6.329 trillion. So while we use growth rates determined using CBO data, the gap does not close.

@martinholmer correctly pointed out that while IRS-SOI statistics are based on only tax filers, CBO projections are for the economy as a whole, which lead to some speculation that the problem is the wages of non-filers are underestimated in the PUF.

However, both @feenberg and @Amy-Xu think the difference may be explained by different definitions of wages and salaries used as well as who is represented in the measurements, rather than just non-filers being underrepresented.

**Comment by martinholmer** [comment]
@andersonfrailey, Thanks for the summary in issue #1156 of the lengthy conversation in #1105.  My view of that conversation is that we got much further than doing "some speculation" about the source of the underestimate.

Here is what you said in #1105 about wages and salaries of filers and non-filers:
> Total wages and salaries for filers in 2014 was $6.784 trillion according to the IRS-SOI data and TaxCalc with the new weights produces $6.819 trillion (up from $6.795 trillion) for filers, so I believe this diagnosis [that the underestimate is caused by non-filers having too little earnings] could be correct because all of the non-filers on the record would be coming from the CPS and they only add $1.3 billion in wages, ...

Then later that same day, you said in response to my statements:
> This statement is also correct:
>
>> So, you are saying that in 2009 before we start any blowup logic, `puf.csv` has $5.848 trillion in wages and salaries while CBO has $6.329 trillion. That means that before any blowup in the years after 2009 that we are 7.6 percent below the CBO number.
>
> With regards to this:
>
>> If we want to be in line with CBO, then we need to fix the massive understatement of non-filer earnings in the `puf.csv` file.
>
> This is how I am understanding the issue. I'd like to talk to John to get his opinion as well, but yes, I believe it is possible the earnings of non-filers are being understated in the puf and we should look into ways to increase them.

Yes, it would be good to get John's opinion, but in order to resolve this issue we need to move beyond opinion.  

Where in the taxdata repo is the control total for non-filer wage-and-salary income?  This question assumes there were control totals to hit in the process of adding CPS records to represent non-filers in the `puf.csv` file.  Is that assumption correct?  Where is the documentation for the process of adding the non-filers from the CPS?

@MattHJensen @Amy-Xu

**Comment by feenberg** [comment]
Can some one point me the the source of the CBO number? URL and page, 
please. I want to look at the description and related numbers. I did find 
a CBO number for wages and salaries for 2016, and it did not include 
self-employment income. So my first thought was probably not correct.

Given that the last official reconciliation of SOI and NIPA:

   https://www.bea.gov/scb/pdf/2007/11%20November/1107_pi_agi.pdf

was for 2005, and showed $14b for wages of non-filers ((1-.89)*149b)
I think we should not impute .5T to non-filers. We should ask the relevant 
person at CBO if they can suggest a reason for the difference. They must 
know something, otherwise their revenue estimate would be inaccurate. It 
appears that Kurt Seibert at CBO does FICA tax estimates, possibly he 
would be the person to ask.

dan

**Comment by andersonfrailey** [comment]
@martinholmer asked:
>Where in the taxdata repo is the control total for non-filer wage-and-salary income? This question assumes there were control totals to hit in the process of adding CPS records to represent non-filers in the puf.csv file. Is that assumption correct? Where is the documentation for the process of adding the non-filers from the CPS?

There is no target for total for non-filer wage and salary income because we don't have a measure for estimating how much total non-filer wage and salary income there is. Non-filers are added during the statistical matching process and not targeted after that.

Documentation for the matching process is not currently on GitHub. I will add that right after posting this. The paper I have is a little outdated, but still gets the point across. Here is the section related to adding non-filers:

>Non-filers. TPC relied on a series of probit equations reported by Cilke (1998) to identify potential CPS tax units that were not legally required to file a federal income tax return but did. Cilke related the probability of filing a return (given that the taxpayer was not legally required to do so) as a function of income and numerous demographic variables. These probabilities were then adjusted to approximate control totals for certain taxpayer groups. Incorporating this methodology, while somewhat time-consuming, is straightforward. Because this would have necessitated rerunning the match to include many more demographic variables, we approximated this approach by randomly selecting non-filers in specific taxpayer subgroups and re-classified them as filers so as to match SOI totals in that subgroup.

@feenberg, The CBO number can be found in the 10-year economic projections. It is from the August 2016 calendar year estimates. The spreadsheet with them can be found here: https://www.cbo.gov/about/products/budget-economic-data#7
And the full paper that goes with it can be found here: https://www.cbo.gov/sites/default/files/114th-congress-2015-2016/reports/51129-2016outlookonecol-2.pdf

I did make one small mistake in my original post. In the CBO projections for wages and salaries I posted fiscal year estimates, not calendar year. These are the calendar year projections:

| Year |         | 
|------|---------| 
| 2016 | 8204.5  | 
| 2017 | 8562.3  | 
| 2018 | 8910.9  | 
| 2019 | 9234.7  | 
| 2020 | 9568.6  | 
| 2021 | 9938.2  | 
| 2022 | 10329.2 | 
| 2023 | 10737.2 | 
| 2024 | 11160.3 |

**Comment by martinholmer** [comment]
@andersonfrailey,  Thanks for the info about the non-filers that included this statement:
> Because this would have necessitated rerunning the match to include many more demographic variables, we approximated this approach by randomly selecting non-filers in specific taxpayer subgroups and re-classified them as filers so as to match SOI totals in that subgroup.

So, what does this mean?  Certain CPS records were added to the `puf.csv` sample because they represent non-filers, but then they were "re-classified ... as filers so as to match SOI totals in that subgroup".  What sense does that make?  SOI doesn't have any information about non-filers.

@MattHJensen @feenberg @Amy-Xu

**Comment by martinholmer** [comment]
@feenberg said in issue #1156:
> Given that the last official reconciliation of SOI and NIPA:
>
>  https://www.bea.gov/scb/pdf/2007/11%20November/1107_pi_agi.pdf
>
> was for 2005, and showed $14b for wages of non-filers ((1-.89)*149b) I think we should not impute .5T to non-filers.

Dan, thanks for the link to the useful BEA report entitled [Comparison of BEA Estimates of Personal Income and IRS Estimates of Adjusted Gross Income](https://www.bea.gov/scb/pdf/2007/11%20November/1107_pi_agi.pdf).

I've looked at this BEA paper and have a slightly different interpretation than what you offer above.  Let me explain.  The paper says it estimates a total "misreporting adjustment [that] comprises a filer adjustment, which measures the amount of income that is underreported on tax returns, and a nonfiler adjustment, which measures the amount of income earned by tax return nonfilers" (p.37).  We agree that the paper says in 2005 "the nonfiler adjustment accounted for ... 11 percent of the [total] wage and salary misreporting adjustment" (p.37).  But it seems to me that in Table 1 (p.38) the total wage and salary misreporting adjustment is 119.4 billion dollars, not the 149 you mention above.  So, my calculations (following the same logic as you used) indicate that in 2005 nonfilers had wage and salary income of 13.134 billion dollars (= 0.11 x 119.4).

Table 1 also states that IRS wage and salary income for filers was 5115.4 billion dollars in 2005.  So, that means that the ratio of nonfiler wage and salary income to filer wage and salary income is about 0.002568 in 2005.

But, as the following calculations show, the 2009 ratio of nonfiler to filer wage and salary income in the `puf.csv` file (before any blowup factors are applied) is far below the BEA-estimated 2005 ratio of 0.002568.  Here are my calculations with the `puf.csv` file:
```
$ cp ../tax-calculator-data/puf.csv .
$ ../csvvars puf.csv | grep filer
71 filer
$ ../csvvars puf.csv | grep s006
70 s006
$ ../csvvars puf.csv | grep e00200
12 e00200
73 e00200p
74 e00200s
$ awk -F, 'NR>1{e[$71]+=$12*$70/100}END{for(f in e)print f,e[f]*1e-9}' puf.csv
0 2.06174
1 5846.26
```
So, in the 2009 `puf.csv` file, the nonfiler to filer wage and salary ratio is about 0.000353 (= 2.06174 / 5846.26), which is roughly one tenth the size of the 2005 BEA-estimated ratio.

If we assume that nothing changed between 2005 and 2009 that would cause this ratio to shift, then we can compute a target wage and salary income for nonfilers in 2009 of about 15.0 billion dollars (= 0.002568 x 5846.26).  The `puf.csv` file has nonfiler  wage and salary income of about 2.06 billion dollars, much less than the target 15.0 billion dollars.  And, of course, this gap grows in dollar terms as the 2009 data are "aged" to future years.

So, if these calculations make sense, they provide a target wage and salary income total for the CPS records added to represent nonfilers.  And this target is independent of any CBO projections.  

It seems to me that the process that adds CPS records into the `puf.csv` file should be revised to include this wage and salary income target and the revised process should be used to create a new, better 2009 `puf.csv` file.

@MattHJensen @Amy-Xu @andersonfrailey @GoFroggyRun @codykallen @zrisher

**Comment by feenberg** [comment]
Comments below.

On Thu, 26 Jan 2017, Martin Holmer wrote:

> 
> @feenberg said in issue #1156:
>
>       Given that the last official reconciliation of SOI and NIPA:
>
>       https://www.bea.gov/scb/pdf/2007/11%20November/1107_pi_agi.pdf
>
>       was for 2005, and showed $14b for wages of non-filers
>       ((1-.89)*149b) I think we should not impute .5T to non-filers.
> 
> Dan, thanks for the link to the useful BEA report entitled Comparison of BEA
> Estimates of Personal Income and IRS Estimates of Adjusted Gross Income.
> 
> I've looked at this BEA paper and have a slightly different interpretation
> than what you offer above. Let me explain. The paper says it estimates a
> total "misreporting adjustment [that] comprises a filer adjustment, which
> measures the amount of income that is underreported on tax returns, and a
> nonfiler adjustment, which measures the amount of income earned by tax
> return nonfilers" (p.37). We agree that the paper says in 2005 "the nonfiler
> adjustment accounted for ... 11 percent of the [total] wage and salary
> misreporting adjustment" (p.37). But it seems to me that in Table 1 (p.38)
> the total wage and salary misreporting adjustment is 119.4 billion dollars,
> not the 149 you mention above. So, my calculations (following the same logic
> as you used) indicate that in 2005 nonfilers had wage and salary income of
> 13.134 billion dollars (= 0.11 x 119.4).
> 
> Table 1 also states that IRS wage and salary income for filers was 5115.4
> billion dollars in 2005. So, that means that the ratio of nonfiler wage and
> salary income to filer wage and salary income is about 0.002568 in 2005.
> 
> But, as the following calculations show, the 2009 ratio of nonfiler to filer
> wage and salary income in the puf.csv file (before any blowup factors are
> applied) is far below the BEA-estimated 2005 ratio of 0.002568. Here are my
> calculations with the puf.csv file:
> 
> $ cp ../tax-calculator-data/puf.csv .
> $ ../csvvars puf.csv | grep filer
> 71 filer
> $ ../csvvars puf.csv | grep s006
> 70 s006
> $ ../csvvars puf.csv | grep e00200
> 12 e00200
> 73 e00200p
> 74 e00200s
> $ awk -F, 'NR>1{e[$71]+=$12*$70/100}END{for(f in e)print f,e[f]*1e-9}' puf.c
> sv
> 0 2.06174
> 1 5846.26
> 
> So, in the 2009 puf.csv file, the nonfiler to filer wage and salary ratio is
> about 0.000353 (= 2.06174 / 5846.26), which is roughly one tenth the size of
> the 2005 BEA-estimated ratio.
> 
> If we assume that nothing changed between 2005 and 2009 that would cause
> this ratio to shift, then we can compute a target wage and salary income for
> nonfilers in 2009 of about 15.0 billion dollars (= 0.002568 x 5846.26). The
> puf.csv file has nonfiler wage and salary income of about 2.06 billion
> dollars, much less than the target 15.0 billion dollars. And, of course,
> this gap grows in dollar terms as the 2009 data are "aged" to future years.
> 
> So, if these calculations make sense, they provide a target wage and salary
> income total for the CPS records added to represent nonfilers. And this
> target is independent of any CBO projections.
> 
> It seems to me that the process that adds CPS records into the puf.csv file
> should be revised to include this wage and salary income target and the
> revised process should be used to create a new, better 2009 puf.csv file.
>

These are excellent calculations, and move us ahead. My next question is 
"How is filing status determined in the CPS?". Do we use the variable 
provided by Census? Do we know how that is determined? I would like to see 
anyone age-eligible for the EIC with wage income classified as a filer, as 
such persons can profit from filing, either by getting the EIC or a return 
of over-withholding. I don't think classifying based on total income is a 
good way to go, if that is how it is now done.

dan

**Comment by andersonfrailey** [comment]
@martinholmer asked:
>So, what does this mean? Certain CPS records were added to the puf.csv sample because they represent non-filers, but then they were "re-classified ... as filers so as to match SOI totals in that subgroup". What sense does that make? SOI doesn't have any information about non-filers.

My interpretation is this: there are units in SOI who legally did not need to file, but did anyway. So they found the probability of a tax unit filing conditional on that unit not being legally required to do so. Based on that probability, some non-filing units in the CPS are randomly re-assigned in order to account for those in the SOI who did not need to file but did anyway.

After this random re-assignment, those tax units still classified as non-filers are kept separate from the filers and not included in the process that matches SOI and CPS records. They are added on in the very last step of the file creation after matched units have been created.

The matching code isn't in the TaxData repo but can be found in TaxData PR [#39](https://github.com/open-source-economics/taxdata/pull/39/files). 

@feenberg asked:
>My next question is
"How is filing status determined in the CPS?". Do we use the variable
provided by Census? Do we know how that is determined?

There are three tests used to determine if a CPS unit is flagged as a filer:

1. Wage test. If anyone in the tax unit has positive wages and salaries income, the unit is deemed to have filed.
2. Gross income test. If the tax unit's income exceeds the thresholds in the form 1040 filing requirements, they're deemed to have filed.
4. Dependent filer test. 

These tests are applied to Tax Units not flagged as non-filers in the CPS.

Beyond those tests, there is also the random assignment I explained in response to @martinholmer.

**Comment by andersonfrailey** [comment]
I also like @martinholmer's idea for adding a target for non-filers to the process. I'd like to note that a switch to a matched `puf.csv` file based on the 2010 PUF and 2016 CPS is in the pipeline, though I can't give a date it will be completed right now. It's not unreasonable to believe this problem with wages and salaries will persist though so some discussion further fleshing out @martinholmer's idea could be useful.

**Comment by feenberg** [comment]
On Thu, 26 Jan 2017, andersonfrailey wrote:

> 
> I also like @martinholmer's idea for adding a target for non-filers to the process. I'd like to note that a
> switch to a matched puf.csv file based on the 2010 PUF and 2016 CPS is in the pipeline, though I can't give
> a date it will be completed right now. It's not unreasonable to believe this problem with wages and salaries
> will persist though so some discussion further fleshing out @martinholmer's idea could be useful.
>

Are we concerned with getting the # of non-filers right, or the dollar 
amounts right? For most non-filers, the dollar amount of income or tax 
will be very small, so errors in the # of non-filers shouldn't be 
material. Would it make more sense to report number with non-zero 
liability rather than filing status?

dan

**Comment by martinholmer** [comment]
@feenberg said:
> Are we concerned with getting the # of non-filers right, or the dollar amounts right?  For most non-filers, the dollar amount of income or tax will be very small, so errors in the # of non-filers shouldn't be material.  Would it make more sense to report number with non-zero liability rather than filing status?

Any nonfiler with earnings is going to have substantial tax liability relative to earnings because of the payroll tax.  It is earnings we're short on.  That seems to me the primary issue is earnings and we have a target for the BEA tabulations I presented earlier in the #1156 conversation.  How can we target the number of nonfilers?  We don't have a target for that do we?

@MattHJensen @Amy-Xu @andersonfrailey

**Comment by feenberg** [comment]
On Thu, 26 Jan 2017, Martin Holmer wrote:

> 
> @feenberg said:
>
>       Are we concerned with getting the # of non-filers right, or the dollar amounts right? For
>       most non-filers, the dollar amount of income or tax will be very small, so errors in the #
>       of non-filers shouldn't be material. Would it make more sense to report number with
>       non-zero liability rather than filing status?
> 
> Any nonfiler with earnings is going to have substantial tax liability relative to earnings because of
> the payroll tax. It is earnings we're short on. That seems to me the primary issue is earnings and we

Are we short on payroll tax aggregate? I haven't seen any claim that our 
estimate of FICA is especially low.

dan

**Comment by andersonfrailey** [comment]
@martinholmer asked:
> How can we target the number of nonfilers? We don't have a target for that do we?

We do not have a target for the number of non-filers. All of the CPS tax units that are determined to not be filers are just added on at the end of the matching process.

@feenberg, here are our payroll tax calculations compared to CBO's:

| Source  | 2015     | 2016     | 2017     | 2018     | 2019     | 2020     | 2021     | 2022     | 2023     | 2024     | 2025     | 2026     | 
|---------|----------|----------|----------|----------|----------|----------|----------|----------|----------|----------|----------|----------| 
| CBO ($B)     | 1065.257 | 1114.074 | 1149.322 | 1187.958 | 1227.546 | 1266.901 | 1314.943 | 1363.73  | 1413.906 | 1465.111 | 1521.493 | 1578.862 | 
| TaxCalc ($B) | 999.509  | 1039.672 | 1088.760 | 1135.865 | 1179.369 | 1224.773 | 1273.148 | 1324.452 | 1379.311 | 1436.128 | 1495.174 | 1556.678 | 



We're a little short, but nothing too drastic. Currently looking for other estimates to compare to.

**Comment by feenberg** [comment]
On Fri, 27 Jan 2017, andersonfrailey wrote:

> 
> @feenberg, here are our payroll tax calculations compared to CBO's:
> 
> Source
> 2015
> 2016
> 2017
> 2018
> 2019
> 2020
> 2021
> 2022
> 2023
> 2024
> 2025
> 2026
> CBO ($B)
> 1065.257
> 1114.074
> 1149.322
> 1187.958
> 1227.546
> 1266.901
> 1314.943
> 1363.73
> 1413.906
> 1465.111
> 1521.493
> 1578.862
> TaxCalc ($B)
> 999.509
> 1039.672
> 1088.760
> 1135.865
> 1179.369
> 1224.773
> 1273.148
> 1324.452
> 1379.311
> 1436.128
> 1495.174
> 1556.678
> 
> We're a little short, but nothing too drastic. Currently looking for other
> estimates to compare to.

The 2015 shortage of 65B of payroll tax corresponds to 425B of payroll - 
isn't that about the same as the descrepency in wages we are concerned 
about?

dan

**Comment by martinholmer** [comment]
@feenberg said about the @andersonfrailey comparison of CBO and Tax-Calculator projections:
> The 2015 shortage of 65B of payroll tax corresponds to 425B of payroll - isn't that about the same as the discrepancy in wages we are concerned about?

Yes.   That is exactly the point.

Let's try to get away from CBO projections for a minute and focus on how Tax-Calculator is doing in terms of projecting from 2009 to 2016, the most recent year for which we have **historical** (that is, actual) data.

In the [January 2017 CBO 10-year budget projections](https://www.cbo.gov/about/products/budget-economic-data#4), they show actual (not projected) fiscal year 2016 individual income tax and payroll tax revenues as follows:
```
Actual FY2016 inctax:   1546 billion dollars
Actual FY2016 paytax:   1115 billion dollars
```
What do we get from the current-version of Tax-Calculator for CY2016?
```
$ python inctax.py puf.csv 2016 --blowup --weights
You loaded data for 2009.
Instantiation of the calculator automatically extrapolated your data to 2016.

$ awk '{i+=$4*$29;p+=$6*$29}END{print i*1e-9,p*1e-9}' puf-16.out-inctax 
1618.54 1039.67
```
(Notice that the CY2016 payroll tax revenue amount is the same as @andersonfrailey cited.)

Comparing the Tax-Calculator CY2016 estimates with actual FY2016 data produces this:
```
inctax   1618.54 / 1546 = 1.047 (Tax-Calculator almost 5% above actual)
paytax   1039.67 / 1115 = 0.932 (Tax-Calculator almost 7% below actual)
```
So, this is the reason for all the discussion about the CPS nonfilers in the `puf.csv` file, and whether or not they have enough wage and salary income.  

One of the main problems in resolving this issue is that we have not had much detailed documentation about how the CPS units who represent nonfilers were added to the `puf.csv` file.

@MattHJensen @Amy-Xu @codykallen

**Comment by feenberg** [comment]
On Fri, 27 Jan 2017, Martin Holmer wrote:

> 
> @feenberg said about the @andersonfrailey comparison of CBO and
> Tax-Calculator projections:
>
>       The 2015 shortage of 65B of payroll tax corresponds to 425B of
>       payroll - isn't that about the same as the discrepancy in wages
>       we are concerned about?
> 
> Yes. That is exactly the point.
> 
> Let's try to get away from CBO projections for a minute and focus on how
> Tax-Calculator is doing in terms of projecting from 2009 to 2016, the most
> recent year for which we have historical (that is, actual) data.
> 
> In the January 2017 CBO 10-year budget projections, they show actual (not
> projected) fiscal year 2016 individual income tax and payroll tax revenues
> as follows:
> 
> Actual FY2016 inctax:   1546 billion dollars
> Actual FY2016 paytax:   1115 billion dollars
> 
> What do we get from the current-version of Tax-Calculator for CY2016?
> 
> $ python inctax.py puf.csv 2016 --blowup --weights
> You loaded data for 2009.
> Instantiation of the calculator automatically extrapolated your data to 2016
> .
> 
> $ awk '{i+=$4*$29;p+=$6*$29}END{print i*1e-9,p*1e-9}' puf-16.out-inctax 
> 1618.54 1039.67
> 
> (Notice that the CY2016 payroll tax revenue amount is the same as
> @andersonfrailey cited.)
> 
> Comparing the Tax-Calculator CY2016 estimates with actual FY2016 data
> produces this:
> 
> inctax   1618.54 / 1546 = 1.047 (Tax-Calculator almost 5% above actual)
> paytax   1039.67 / 1115 = 0.932 (Tax-Calculator almost 7% below actual)
> 
> So, this is the reason for all the discussion about the CPS nonfilers in the
> puf.csv file, and whether or not they have enough wage and salary income.
> 
> One of the main problems in resolving this issue is that we have not had
> much detailed documentation about how the CPS units who represent nonfilers
> were added to the puf.csv file.

Is it possible to go back to a year for which there is SOI data on AGI, 
wages, EIC, etc? I can see that if we miss a lot of low-wage workers it 
would increase our estimate income tax (because of the EIC) and decrease 
our estimate of FICA, but I would hope to see some confirmation of that in 
the amount of EIC and FICA before acting on that theory.

dan

**Comment by andersonfrailey** [comment]
@feenberg 
I found [SOI collections data for 2014](https://www.irs.gov/uac/soi-tax-stats-collections-and-refunds-by-type-of-tax-irs-data-book-table-1) (thousands of dollars):

|              |            | 
|--------------|------------| 
| Income Tax   | 1,614,213,172 | 
| FICA         | 897,978,551  | 
| SECA         | 63,679,295   | 
| OASDHI Total | 961,657,846  | 


TaxCalc gives the following for income and payroll taxes in 2014 (thousands of dollars):
Income Tax: 1,433,052,034
Payroll Tax: 958,296,326

I haven't found anything from SOI on EIC yet.

**Comment by feenberg** [comment]
On Fri, 27 Jan 2017, andersonfrailey wrote:

> 
> @feenberg
> I found SOI collections data for 2014 (thousands of dollars):

Collections isn't the same as liabilities. Here:

    https://www.irs.gov/uac/soi-tax-stats-individual-income-tax-returns-publication-1304-complete-report#_pt3

there is a column "Total Liabilities" that is 1,444B for 2014. You don't 
say where your number came from. There are lots of different definitions 
of tax - before or after refundable and non-refundable credits, FY v CY, 
etc. What does CBO have for 2014? If we keep to 2014, we can make a more 
rigorous comparison.

> 
> Income Tax
> 1,614,213,172
> FICA
> 897,978,551
> SECA
> 63,679,295
> OASDHI Total
> 961,657,846
> 
> TaxCalc gives the following for income and payroll taxes in 2014 (thousands
> of dollars):
> Income Tax: 1,433,052,034
> Payroll Tax: 958,296,326

If we are low on income tax and high on payroll tax, that is inconsistent 
with missing non-filer income.

dan

**Comment by martinholmer** [comment]
Should leave issue #1156 open as long as the closely-related issue #1549 is open.

**Comment by martinholmer** [comment]
This issue is being closed for the reasons discussed in [this comment](https://github.com/open-source-economics/Tax-Calculator/issues/1549#issuecomment-419065503) to closed issue #1549.
